week2/homework/src/actions/markTodoAsDone.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+const chalk = require('chalk');
+const listTodos = require('./listTodos');
+const storeTodos = require('./storeTodos');
+
+function markTodoAsDone(userIndexInput, allTodos) {
+  if (userIndexInput > allTodos.length || userIndexInput < 1) {
```

Comment:
Since you're doing this check in several places, maybe you could have placed it in a separate function.